### DISCUSSION TITLE
Trouble registering the container as secondary device

### DISCUSSION THREAD

**Discussion by Qhilm** [bbernhard/signal-cli-rest-api#588] [Q&A]
I am able to scan the QR code, but on the iOS app, after accepting, it simply closes the camera without any message of success, and the container is not listed as paired device.

Trying to send any command over curl returns "error: User +12345678 is not registered."

I have enabled debug according to [this](https://github.com/bbernhard/signal-cli-rest-api/blob/master/doc/DEBUG.md) but it doesn't produce any additional docker logs when pairing I have the impression.

Does someone have other ideas for me to gather more logs?

**Comment by bbernhard** [comment]
You could give this version a try: `bbernhard/signal-cli-rest-api:0.167-dev`. It uses the latest signal-cli version (which fixes some bugs regarding the registration). Unfortunately it contains some other issues, that's why I'd suggest to use it only for the registration procedure. After it is registered, I'd switch back to the release version

**Reply by Qhilm** [reply]
Thanks, that fixed the issue!

Then I hit "Error in the HTTP2 framing layer" header but I found that this is a curl issue affecting the version running in the latest macOS (sequoia, curl 8.7.1).

After upgrading to 8.9.1 using brew, this problem also went away.
